apps/web/src/pages/api/sentry/webhook.ts

```diff
@@ -0,0 +1,26 @@
+import { NextApiRequest, NextApiResponse } from 'next';
+import TelegramBot from 'node-telegram-bot-api';
+
+const botToken = process.env.TELEGRAM_API_KEY;
+const chatId = process.env.TELEGRAM_CHAT_ID;
+const bot = new TelegramBot(botToken as string, { polling: false });
+
+const sentryWebhookHandler = async (
+  req: NextApiRequest,
+  res: NextApiResponse
+): Promise<void> => {
+  if (req.method === 'POST') {
```

Comment:
We need to verify signature so we know the payload hasn't been tampered with.
It acts auth: https://docs.sentry.io/product/integrations/integration-platform/webhooks/#verifying-the-signature